src/storybook-components/_data/demoData.js

```diff
@@ -1591,40 +1591,14 @@ const demoData = {
             placeholder: 'Select multiple topics',
             values: [{value: 'topic_a', name: 'Topic A'}, {value: 'topic_b', name: 'Topic B'}, {value: 'topic_c', name: 'Topic C'}, {value: 'topic_d', name: 'Topic D'}]
           },
-
-          {
-            type: 'rplchecklist',
-            label: 'Multi-select drop down',
-            validator (value) {
-              if (Array.isArray(value) && value.length > 0) {
-                return []
-              }
-              return ['Add a selection']
-            },
-            min: 1,
-            required: true,
-            model: 'checklistdropdown',
-            hint: 'Implemented using rplchecklist',
-            placeholder: 'Select multiple topics',
-            values: [{value: 'topic_a', name: 'Topic A'}, {value: 'topic_b', name: 'Topic B'}, {value: 'topic_c', name: 'Topic C'}, {value: 'topic_d', name: 'Topic D'}]
-          },
-
           {
             type: 'rplselect',
             model: 'select',
             required: true,
             validator: ['required'],
             label: 'Single-select drop down',
-            hint: 'Implemented using vue-multiselect',
             placeholder: 'Select a single topic',
-            selectOptions: {
-              trackBy: 'id',
-              label: 'name',
-              closeOnSelect: true,
-              searchable: false,
-              showLabels: false
-            },
-            values: [{id: 'topic_a', name: 'Topic A'}, {id: 'topic_b', name: 'Topic B'}, {id: 'topic_c', name: 'Topic C'}, {id: 'topic_d', name: 'Topic D'}]
+            values: [{id: 'topic_a', name: 'Topic A'}, {id: 'topic_b', name: 'Topic B'}, {id: 'topic_c', name: 'Topic C'}, {id: 'topic_d', name: 'Topic D'}, {id: 'topic_e', name: 'Topic e'}, {id: 'topic_f', name: 'Topic f'}, {id: 'topic_g', name: 'Topic g'}, {id: 'topic_h', name: 'Topic h'}]
```

Comment:
This `rplselect` changes also will bring breaking changes, https://github.com/search?q=org%3Adpc-sdp+rplselect&type=Code
We may can provide a new one to deprecate the old one, something like `rplselectV2`.

Reply:
I dont think we will need to do that, because the external interface is (relatively) the same. Implementers will just need to remove `selectOptions` key, which won't do anything anyway. We could support using `selectOptions.multiple` as well as `multiselect` if it was used alot, but as this was only recently added, I can't see it being used widely.

If vue-multiselect is needed, its available as part of vue-form-generator.